src/evict/evict_lru.c

```diff
@@ -801,18 +801,21 @@ __evict_update_work(WT_SESSION_IMPL *session, bool *eviction_needed)
      *
      * There's an experimental flag WT_CACHE_EVICT_SCRUB_UNDER_TARGET that can be turned on to
      * enable scrub eviction as long as cache usage overall is under half way to the trigger limit.
+     * If the scrub under target config is set, scrub whenever we are under the clean target.
+     * Otherwise, default behavior is to scrub only if we are also under the dirty and update
+     * targets.
      */
     if (__wt_conn_is_disagg(session) && bytes_inuse < (uint64_t)(trigger * bytes_max) / 100)
         LF_SET(WT_EVICT_CACHE_SCRUB);
     else if (bytes_inuse < (uint64_t)((target + trigger) * bytes_max) / 200) {
         if (F_ISSET_ATOMIC_32(
-              &(conn->cache->cache_eviction_controls), WT_CACHE_EVICT_SCRUB_UNDER_TARGET)) {
+              &(conn->cache->cache_eviction_controls), WT_CACHE_EVICT_SCRUB_UNDER_TARGET))
             LF_SET(WT_EVICT_CACHE_SCRUB);
-        } else if (bytes_dirty < (uint64_t)((dirty_target + dirty_trigger) * bytes_max) / 200 &&
-          bytes_updates < (uint64_t)((updates_target + updates_trigger) * bytes_max) / 200) {
+        else if (bytes_dirty < (uint64_t)((dirty_target + dirty_trigger) * bytes_max) / 200 &&
+          bytes_updates < (uint64_t)((updates_target + updates_trigger) * bytes_max) / 200)
             LF_SET(WT_EVICT_CACHE_SCRUB);
-        }
-
+        else
```

Comment:
This change is same as earlier, before the flag code went in. We do not want to keep the pages that are read in the cache if we are above the half way of clean trigger only. If we have a lot of cache, there is no need to evict them urgently.